pandas/tests/frame/test_analytics.py

```diff
@@ -2766,3 +2766,84 @@ def test_multiindex_column_lookup(self):
         result = df.nlargest(3, ("x", "b"))
         expected = df.iloc[[3, 2, 1]]
         tm.assert_frame_equal(result, expected)
+
+    def test_data_frame_value_counts(self):
```

Comment:
Can you split up this test? Roughly one test per "thing" you're testing (single column, raising for unsupported keyword, etc.)